.lando.yml

```diff
@@ -0,0 +1,75 @@
+name: simplytest-me
+recipe: drupal9
+
+config:
+  php: "7.3"
+  webroot: web
+  xdebug: false
+  database: mariadb:10.3
+
+events:
+  # Clear cache after a database import
+  post-db-import:
+    - appserver: cd $LANDO_WEBROOT && drush cache-rebuild --yes
+
+services:
+  appserver:
+    run:
+      - cd $LANDO_MOUNT && composer install --no-scripts
+
+  nodejs:
+    type: node:14
+    globals:
+      gulp-cli: latest
+    run:
+      - cd $LANDO_MOUNT/web/profiles/contrib/simplytest/themes/simplytest_theme && npm install
+      - cd $LANDO_MOUNT/web/profiles/contrib/simplytest/themes/simplytest_theme && npm run build
+
+  phpmyadmin:
+    type: phpmyadmin
+
+tooling:
+  drush:
+    service: appserver
+    env:
+      DRUSH_OPTIONS_URI: "https://simplytest-me.lndo.site/"
+  compile:
+    service: nodejs
+    cmd: cd $LANDO_MOUNT/web/profiles/contrib/simplytest/themes/simplytest_theme && npm run build
+  gulp:
+    service: nodejs
+  node:
+    service: nodejs
+  npm:
+    service: nodejs
+  phpunit:
+    service: appserver
+    cmd: vendor/bin/phpunit
+  phpcs:
+    service: appserver
+    cmd: vendor/bin/phpcs
+  security-check:
+    service: appserver
+    cmd: "php vendor/sensiolabs/security-checker/security-checker security:check composer.lock"
+  si:
+    service: appserver
+    cmd: $LANDO_MOUNT/.ddev/commands/web/si
+    env:
+      DRUSH_OPTIONS_URI: "https://simplytest-me.lndo.site/"
```

Comment:
Do we need to mount this command?

Reply:
Couldn't `cmd` just be

```
 drush si simplytest --existing-config --account-pass=admin --yes
```

Reply:
@mglaman I believe that @benjifisher mounted this command so that the instructions for DDEV and Lando could be very similar in the README:

# Installation with DDEV
 - Clone this repo.
 - Run `composer install`
 - Run `ddev si`

# Installation with Lando
 - Clone this repo.
 - Run `lando start`
 - Run `lando si`

If you disagree, feel free to edit the README to include the longer `drush si` command.

Reply:
Let me make a PR code suggestion, I was just saying it doesn't need to call the ddev command file

Reply:
If anything, I would be more inclined to add `composer install` to DDEV's `config.yml` so it installs on start:

```yaml
hooks:
  post-start:
    # Install project dependencies
    - composer: install
```

Then the instructions would be nearly identical: `ddev start && ddev si` or `lando start && lando si`.

However, I note your suggestion above to execute this only on `build` and not on each `run`, which seems sensible.

Reply:
I guess it's a matter of preference. But I don't think every time you start the project that dependencies need to fire. DDEV doesn't have a relevant `build` command. That's fine. Lando does. I personally like that, I really dislike `lando rebuild` doing a bunch of dependencies installs when all I need is a container rebuild.

Reply:
I think we're over-engineering a helper command that installs the site. Especially if we get the profile to contain the config and define `config_install: true`.

Reply:
It looks as though the change has already been made:

```
    cmd: drush si simplytest --existing-config --account-pass=admin --yes
```